index.html

```diff
@@ -1528,6 +1532,26 @@ <h2>Relationship to Verifiable Credentials</h2>
 is the controller of the <a>verification method</a> used to verify
 the proof.
         </p>
+
+        <p>
+Document authors and implementers are advised to understand the difference
+between the validity period of a <a href="#proofs">proof</a>, which is expressed
+using the <a href="#dfn-created">`created`</a> and <a
+href="#dfn-expires">`expires`</a> properties, and the validity period of a
+<a>verifiable credential</a>, which is expressed using the
```

Comment:
replace <a>verifiable credential</a> with
 credential object

We have already agreed that there are 3 entities of interest:
1. the underlying physical certificate/qualification/credential
2. the credential object which is the JSON representation of 1.
3. the verifiable credential.
Thus the validFrom and validUntil properties will have been taken from the validity period of 1.

Reply:
> [@David-Chadwick] We have already agreed that there are 3 entities of interest:

*You* have agreed upon this.

You have a consistent pattern of stating that things you have asserted have been agreed upon by the rest of the group. Such blind and repetitive statements _can_ be a powerful rhetorical tool, but they are _not_ a good way to proceed when trying to develop a globally acceptable and ratified standard.

> 1. the underlying physical certificate/qualification/credential

There is absolutely no requirement that your first listed item must exist. Such a physical document *might* exist but it equally might *not*.

As an issuer, my assertions may be made and placed directly in your second listed item, for protection/encapsulation by/in your third listed item.

> 2. the credential object which is the JSON representation of 1.

Not JSON, JSON-LD. We *have* all agreed on JSON-LD representation (i.e., `application/vc+ld+json`), as of the Miami F2F.

And, as noted above, your item 1 may not exist, so item 2 is a JSON-LD representation of some number of assertions, prepared for protection/encapsulation in/by item 3.

> 3. the verifiable credential.

Being item 2 plus proof attachment/insertion.

> Thus the `validFrom` and `validUntil` properties will have been taken from the validity period of 1.

Except that item 1 may not exist, so may not provide any validity period details.

Reply:
I do not see how your statements above negate any of my original ones. The fact that there often is a physical certificate, and that this is an entity of interest, does not state that there is a requirement for it to always exist. The fact that JSON-LD is a subclass of JSON does not negate that fact that the credential is also JSON. Finally PR#1211 from the data model states that the validity period is optional, and if present, it is the validity period of the credential and not of the verifiableCredential.

Reply:
@David-Chadwick, I believe what @TallTed is saying is the following:

> the underlying physical certificate/qualification/credential

This thing might not exist, so asserting that the validity period of the proof, or the validity period of the verifiable credential lines up with it (the physical credential) is not always true, and might become increasingly not true because some physical credentials might cease to be issued because their digital counterparts are easier to issue/manage.

So, tying any validity period to the physical certificate is fraught.

> the credential object which is the JSON representation of 1.

While this thing exists, the group tends to not care as much about it as the verifiable credential. So, stating that the `validFrom` and `validUntil` refers to the credential object isn't quite right either... because both of those items are optional and what makes them enforceable (in a meaningful way) is their expression on a verifiable credential.

> the verifiable credential. Thus the validFrom and validUntil properties will have been taken from the validity period of 1.

Not necessarily... which is the point. I think you're wanting to say something that we don't have consensus on.

The only thing this PR is attempting to highlight is that the validity period of the proof might not always line up with the validity period of the verifiable credential... and that's it. The validity period of the "credential object" (as you've coined the term) is a bit murky because the thing isn't secured... so is it really the validity period, or just something some attacker injected into the "credential object"?

I'd like to merge this PR... can you think of any language we could add that would address your concern AND not raise objections from others that have reviewed this PR?

Reply:
Please take another look at  PR#1211 from the data model (which I think you authored). This states that the validity period is optional, and if present, it is the validity period of the credential. (You can ignore my addition of the word object to this, since most (all?) of our data structures can be regarded as objects). It does not say that it is the validity period of the verifiable credential, which you are stating above. So clearly there is a disconnect somewhere that needs fixing. This is my concern.

Reply:
@msporny said "The validity period of the "credential object" (as you've coined the term)". I did not coin this term. It is already in the VCDM quote "For example, a credential  object with the type of ExampleDegreeCredential ," :-)

Reply:
@David-Chadwick wrote:
>  I did not coin this term. It is already in the VCDM quote "For example, a credential object with the type of ExampleDegreeCredential ,"

Hmm... I'm not seeing the term "credential object" here: https://github.com/w3c/vc-data-model/pull/1211/files

Ah!, but I do now see it in the main spec:

https://w3c.github.io/vc-data-model/

... once... and I believe that to be an error. So, you're right, the term /does/ exist in the main spec, but since it only appears once, it's probably just a hold over from previous iterations of the spec. 

And, as you say... saying "object" is probably neither here nor there because many of the things in the spec that we describe could be viewed as "objects".

I /thought/ that you were describing a NEW kind of thing, that you were calling a "credential object", which was purely JSON (and unsecured). I've re-read your statements in this thread and now am thinking that you are asserting that "credential" as described in the main spec and "credential object" are synonymous. They are exactly the same thing. Thus, we can just simplify your statements above to "credential", as described by the specification... and the thing that differentiates a "credential" from a "verifiable credential" is the fact that the latter is secured in some way, and the former is not?

> This states that the validity period is optional, and if present, it is the validity period of the credential. (You can ignore my addition of the word object to this, since most (all?) of our data structures can be regarded as objects). It does not say that it is the validity period of the verifiable credential, which you are stating above. So clearly there is a disconnect somewhere that needs fixing. This is my concern.

I had to go and look at the main spec again, and yes, you are correct. If we change this language to align with the main spec, would that address your concern on this PR, @David-Chadwick ? That is, we change the mention of "validFrom" and "validUntil" relationship from "verifiable credential" to "credential" in this PR. Would that address your concern?

Reply:
Yes and I produced the revised text for you below (2 days ago)

Reply:
I see it now, ok, thx @David-Chadwick.